service/project.go

```diff
@@ -89,7 +89,7 @@ func (uis *UIServer) projectPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	projVars, err := model.FindOneProjectVars(id)
+	projVars, err := model.FindMergedProjectVars(id)
```

Comment:
Because this is for the legacy UI, we only ever want to display the branch-level variables.